resource_transip_private_network_attachment_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+
+	"os"
+	"testing"
+)
+
+func TestAccTransipResourcePrivateNetworkAttachment(t *testing.T) {
+	testConfig := fmt.Sprintf(`
+	resource "transip_private_network_attachment" "test" {
+		private_network_id = "%s"
+		vps_id = "%s"
+	}
+	`, os.Getenv("TF_VAR_private_network_name"), os.Getenv("TF_VAR_vps_name"))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config:        testConfig,
+				ResourceName:  "transip_private_network_attachment.test",
+				ImportState:   false,
+				ImportStateId: os.Getenv("TF_VAR_private_network_name"),
```

Comment:
When using an environment variable in multiple places, it might be better to put it in a variable. This reduces the risk that a typo occurs.

Reply:
I agree, it also looks nicer as well. Have been a bit inconsistent on this, will fix.